zipkin2/src/main/java/zipkin2/Span.java

```diff
@@ -243,26 +254,25 @@ public Builder toBuilder() {
     String traceId, parentId, id;
     Kind kind;
     String name;
-    Long timestamp, duration;
+    long timestamp, duration;
     Endpoint localEndpoint, remoteEndpoint;
     ArrayList<Annotation> annotations;
     TreeMap<String, String> tags;
-    Boolean debug, shared;
+    int flags = 0;
```

Comment:
ps I recall a conversation with @swankjesse that in java an int is better than a byte even if you only need a few flags. That the usage is easier and that it behaves like an int anyway in memory, iirc. correct me if wrong